<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">FLibrary</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="settingsLineEdit">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QSplitter" name="verticalSplitter">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="handleWidth">
              <number>0</number>
             </property>
             <widget class="QWidget" name="navigationWidget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
             <widget class="QWidget" name="rightWidget" native="true">
              <layout class="QGridLayout" name="gridLayout_2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QSplitter" name="horizontalSplitter">
                 <property name="lineWidth">
                  <number>0</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="handleWidth">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="booksWidget" native="true">
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="annotationWidget" native="true">
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="logPage">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListView" name="logView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_User_data">
     <property name="title">
      <string>&amp;User data</string>
     </property>
     <addaction name="actionExportUserData"/>
     <addaction name="actionImportUserData"/>
    </widget>
    <addaction name="menu_User_data"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuLog">
     <property name="title">
      <string>&amp;Log</string>
     </property>
     <widget class="QMenu" name="menuLogVerbosityLevel">
      <property name="title">
       <string>Log &amp;verbosity level</string>
      </property>
     </widget>
     <addaction name="actionShowLog"/>
     <addaction name="actionClearLog"/>
     <addaction name="actionShowCollectionStatistics"/>
     <addaction name="menuLogVerbosityLevel"/>
    </widget>
    <widget class="QMenu" name="menuAnnotation">
     <property name="title">
      <string>&amp;Annotation</string>
     </property>
     <addaction name="actionHideAnnotation"/>
     <addaction name="actionShowAnnotationCover"/>
     <addaction name="actionHideAnnotationCover"/>
     <addaction name="actionShowAnnotationContent"/>
     <addaction name="actionHideAnnotationContent"/>
    </widget>
    <addaction name="actionShowRemoved"/>
    <addaction name="actionHideRemoved"/>
    <addaction name="menuAnnotation"/>
    <addaction name="actionShowAnnotation"/>
    <addaction name="actionShowStatusBar"/>
    <addaction name="actionHideStatusBar"/>
    <addaction name="menuLog"/>
   </widget>
   <widget class="QMenu" name="menuCollection">
    <property name="title">
     <string>&amp;Collection</string>
    </property>
    <widget class="QMenu" name="menuSelectCollection">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Select &amp;collection</string>
     </property>
    </widget>
    <addaction name="actionAddNewCollection"/>
    <addaction name="menuSelectCollection"/>
    <addaction name="actionRemoveCollection"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>&amp;Language</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuFont">
     <property name="title">
      <string>&amp;Font</string>
     </property>
     <addaction name="actionFontSettings"/>
     <addaction name="actionFontSizeUp"/>
     <addaction name="actionFontSizeDown"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="menuFont"/>
    <addaction name="actionScripts"/>
    <addaction name="actionExportTempate"/>
    <addaction name="actionPermanentLanguageFilter"/>
    <addaction name="separator"/>
    <addaction name="actionRestoreDefaultSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionCheckForUpdates"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuBook">
    <property name="title">
     <string>&amp;Book</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuCollection"/>
   <addaction name="menuBook"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFontSizeUp">
   <property name="text">
    <string>Size &amp;Up</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Up</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionFontSizeDown">
   <property name="text">
    <string>Size &amp;Down</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Down</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string notr="true">Alt+X</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAddNewCollection">
   <property name="text">
    <string>Add &amp;new collection...</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionRemoveCollection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Remove collection</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionHideRemoved">
   <property name="text">
    <string>&amp;Hide removed books</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionShowLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show log</string>
   </property>
   <property name="shortcut">
    <string notr="true">Esc</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionClearLog">
   <property name="text">
    <string>&amp;Clear log</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionShowCollectionStatistics">
   <property name="text">
    <string>Show collection s&amp;tatistics</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionFontSettings">
   <property name="text">
    <string>Font &amp;settings...</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionExportUserData">
   <property name="text">
    <string>E&amp;xport</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionImportUserData">
   <property name="text">
    <string>I&amp;mport</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionShowRemoved">
   <property name="text">
    <string>&amp;Show removed books</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionRestoreDefaultSettings">
   <property name="text">
    <string>&amp;Restore default values</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionHideAnnotation">
   <property name="text">
    <string>&amp;Hide</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionShowAnnotation">
   <property name="text">
    <string>Show &amp;annotation</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionShowAnnotationContent">
   <property name="text">
    <string>Show &amp;content</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionHideAnnotationContent">
   <property name="text">
    <string>Hide &amp;content</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionShowStatusBar">
   <property name="text">
    <string>Show s&amp;tatus line</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionHideStatusBar">
   <property name="text">
    <string>Hide s&amp;tatus line</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionShowAnnotationCover">
   <property name="text">
    <string>Show co&amp;ver</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionHideAnnotationCover">
   <property name="text">
    <string>Hide co&amp;ver</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionTestLogColors">
   <property name="text">
    <string>&amp;Test colors</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionScripts">
   <property name="text">
    <string>&amp;Scripts...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionExportTempate">
   <property name="text">
    <string>Book &amp;export tempate</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCheckForUpdates">
   <property name="text">
    <string>Check for &amp;Updates</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionPermanentLanguageFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Permanent language filter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/gui.qrc"/>
 </resources>
 <connections/>
</ui>
